docs/source/dev/profiling/profiling_index.rst

```diff
@@ -0,0 +1,33 @@
+Profiling vLLM 
+=================================
+
+We support tracing vLLM workers using the ``torch.profiler`` module. You can enable tracing by setting the ``VLLM_TORCH_PROFILER_DIR`` environment variable to the directory where you want to save the traces: ``VLLM_TORCH_PROFILER_DIR=/mnt/traces/``
```

Comment:
Suggest to use a more common path as an example, such as `$HOME/traces/` or `/tmp/traces`